configs/rpt/simpleusb.conf

```diff
@@ -116,6 +116,11 @@ clipledgpio = 1                     ; Enable ADC Clip Detect feature to use a GP
                                     ; Supports URI Clip LED by setting a GPIO (if available) high for 500mS
                                     ; when clipping detected. Value = GPIO# to use (GPIO1 recommended)
 
+legacyaudioscaling = no             ; If yes, continue to do raw audio sample scaling and clipping, resulting in Tx audio levels increasing
+                                    ; by 0.78dB and Rx audio levels increasing by 0-1.5dB. This should be set to no unless you have an
+                                    ; existing node with precisely adjusted audio levels but are unable to adjust them. If set to
+                                    ; yes, degraded audio quality will result
```

Comment:
but are unable --> and are unable

audio quality will --> audio quality may

Reply:
updated.